docs/apache-airflow/redirects.txt

```diff
@@ -15,68 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# New URL for Google Guide index
-howto/operator/gcp/index.rst howto/operator/google/index.rst
-# New URLs for Google Guides
-howto/operator/gcp/ads.rst howto/operator/google/ads.rst
```

Comment:
What happens if someone navigates to `/docs/stable/howto/operator/google/ads.html` -- can we redirect that to the stable version of the new provider?

Reply:
> In Airflow 2.0, we are missing a few more files, but I will redirect them with .htaccess, because these files are moved between different documentation packages.

I'm working on it now.

Reply:
<img width="1382" alt="Screenshot 2020-12-09 at 14 52 18" src="https://user-images.githubusercontent.com/12058428/101638283-28105e00-3a2e-11eb-8634-cc56efba65f9.png">

Reply:
Nice one!

Reply:
PR: https://github.com/apache/airflow-site/pull/341